dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -824,7 +824,7 @@ class PerfTestWithSkSL extends PerfTest {
       await _generateSkSL();
 
       // Build the app with SkSL artifacts and run that app
-      final String observatoryUri = await _buildAndRun();
+      final String observatoryUri = await _runApp(skslPath: _skslJsonFileName);
```

Comment:
`appBinary` doesn't need to be passed in anymore?

Reply:
Instead of building the apk and then running the prebuilt app to work around run not accepting --bundle-sksl-path, we can now just pass the arg to run and have it build the correct app